lattice/scripts/persistence_in_memory.py

```
@@ -0,0 +1,439 @@
+import requests
+
+ROOT = "https://three.arcprize.org"
+AGENT = "the persistence in memory"
+LEARNED_FROM = "https://arcprize.org/scorecards/8aaeff26-b817-4d32-b4cd-e4b9251370b1"
+SOURCE_URL = "https://github.com/drQedwards/pmll"
+DEADLINE_SEC = 13 * 60 + 20
+MIN_INTERVAL = 0.13
+MAX_ACTIONS_BASE = 140
+MAX_ACTIONS_BONUS = 100
+STALL_RESET_AFTER = 42
+OUT = Path("/tmp/arc-persistence")
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/arc-persistence"

(S108)